### PR TITLE
Fix svg paths

### PR DESCRIPTION
**Pull request by jessrosenfield**
[ch313](https://app.clubhouse.io/coprocure/story/313/powered-by-coprocure-logo-broken)

### Summary
Following the migration to the django site, the absolute
paths to our assets have changed, causing the CoProcure
logo to be broken in our search widget.

### Testing
Tested manually that logo was visible locally
with `localserver-components.js` script.
Unable to find test data for manual verification of warning svg.

### CODE REVIEW

File: coverage/base.css [hunk 1]
```diff
@@ -172,16 +172,17 @@ table.coverage td span.cline-any {
 /* dark green */
 .status-line.high, .high .cover-fill { background:rgb(77,146,33) }
 .high .chart { border:1px solid rgb(77,146,33) }
-
-.medium .chart { border:1px solid #666; }
-.medium .cover-fill { background: #666; }
+/* dark yellow (gold) */
```
Comment (achen530) [inline coverage/base.css]: Sorry, what are these for?

File: src/search/constants.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const COPROCURE_STATIC_PATH = "https://www.coprocure.us/static/";
```
Comment (achen530) [inline src/search/constants.js]: minor nit: spaces around plus sign please

File: coverage/sorter.js [hunk 3]
```diff
@@ -161,7 +161,7 @@ var addSorting = (function() {
             return;
         }
         cols = loadColumns();
-        loadData(cols);
+        loadData();
```
Comment (achen530) [inline coverage/sorter.js]: and this?

**Review by achen530 — APPROVED** [review]
I think you're linking to the wrong story in clubhouse so please fix before landing (otherwise you'll auto-complete the other task)

The warning icon is used on contracts that are expired or non-cooperative. If you run a query (my default is to use "test" so I don't pollute the search query data too much =)) and check "Include expired contracts" on the left, you'll see the broken icon.

**Comment by achen530** [comment]
I also included CoProcure/engineers as a reviewer (please do that in the future for visibility). Thanks!

File: src/search/constants.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+export const LOGO_PATH = COPROCURE_STATIC_PATH+"img/logo-svg.svg";
```
Comment (Dehorser) [inline src/search/constants.js]: The Javascript `URL` object might be preferable, in case there are clashing slashes.

**Comment by achen530** [comment]
Oh!! This is in the coprocure-search-widget repo. This fix also needs to be made in coprocure.us-django. You can see the broken icons for the warning icon here:

Screen Shot 2020-04-21 at 2 18 27 PM

File: coverage/base.css (hunk 1 above)
Comment (jessrosenfield) [reply coverage/base.css]: I actually wanted to ask you about this... it looks like I somehow pulled in these changes when following the README instructions for coprocure-search-widget and components.coprocure.us. I wasn't sure whether these should be included or not, so I added them in the "update content from testing" diff so I can easily reset them.

File: coverage/sorter.js (hunk 3 above)
Comment (jessrosenfield) [reply coverage/sorter.js]: Looped in w/  839c511. Thought you might have insight as to how changes to the `coverage` dir were automatically generated.

**Comment by jessrosenfield** [comment]
Per offline discussion I removed the coverage directory from git tracking. In addition, due to lack of compatibility with IE using the `URL` function and recent duplication of the widget into the django repo, I abandoned the `constants.js` refactor in favor of fixing each path directly. See https://github.com/CoProcure/components.coprocure.us/pull/9 for associated updates to the components repo.

**Comment by jessrosenfield** [comment]
> Oh!! This is in the coprocure-search-widget repo. This fix also needs to be made in coprocure.us-django. You can see the broken icons for the warning icon here:
> 
> Screen Shot 2020-04-21 at 2 18 27 PM

Opened https://github.com/CoProcure/coprocure.us-django/pull/26 for fixes to the django repo.

**Pull request merged**